### PR TITLE
Remove behat structure

### PR DESCRIPTION
**Pull request by ahocquard**
**Description (for Contributor and Core Developer)**
CI green (I just renamed feature files)

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: tests/legacy/features/pim/structure/association-type/edit_association_type.feature [hunk 1]
```diff
@@ -1,33 +0,0 @@
...
-    And I am logged in as "Julia"
-
-  Scenario: Successfully edit an association type
-    Given I am on the "SUBSTITUTION" association type page
-    Then I should see the Code field
-    And the field Code should be disabled
-    When I fill in the following information:
-      | English (United States) | My substitution |
-    And I press the "Save" button
-    Then I should see the text "My substitution"
```
Comment (fitn) [inline tests/legacy/features/pim/structure/association-type/edit_association_type.feature]: I'd have keep this one for the edition?

File: tests/legacy/features/pim/structure/attribute/option/edit_attribute_options.feature [hunk 2]
```diff
@@ -1,68 +0,0 @@
-@javascript
-Feature: Edit attribute options
-  In order to define choices for a choice attribute
-  As a product manager
-  I need to edit options for attributes of type "Multi select" and "Simple select"
-
```
Comment (fitn) [inline tests/legacy/features/pim/structure/attribute/option/edit_attribute_options.feature]: I'd have keep one there

File: tests/legacy/features/pim/structure/attribute/option/import_options.feature [hunk 3]
```diff
@@ -24,27 +24,6 @@ Feature: Import options
       | brand     | Nike        | Nike        |
       | brand     | Caterpillar | Caterpillar |
```
Comment (fitn) [inline tests/legacy/features/pim/structure/attribute/option/import_options.feature]: If possible rename this file as import_attribute_options_csv.feature

File: tests/legacy/features/pim/structure/attribute/option/import_options.feature [hunk 4]
```diff
@@ -24,27 +24,6 @@ Feature: Import options
...
-    And the following job "csv_footwear_option_import" configuration:
-      | filePath | %file to import% |
-    When I am on the "csv_footwear_option_import" import job page
-    And I launch the import job
-    And I wait for the "csv_footwear_option_import" job to finish
-    Then I should see the text "skipped 1"
-    And I should see the text "code: This value should not be blank"
-
   @javascript
   Scenario: Skip options with unknown attribute code
```
Comment (fitn) [inline tests/legacy/features/pim/structure/attribute/option/import_options.feature]: I'd have removed this one.

File: tests/legacy/features/pim/structure/association-type/edit_association_type.feature (hunk 1 above)
Comment (ahocquard) [reply tests/legacy/features/pim/structure/association-type/edit_association_type.feature]: there is almost no use case: you can only edit a label

File: tests/legacy/features/pim/structure/attribute/option/import_options.feature (hunk 4 above)
Comment (ahocquard) [reply tests/legacy/features/pim/structure/attribute/option/import_options.feature]: it just checks that we correctly skips lines with error (+ testing one validation message). I think it's still useful to keep it for safety.

File: tests/legacy/features/pim/structure/attribute/option/import_options.feature (hunk 3 above)
Comment (ahocquard) [reply tests/legacy/features/pim/structure/attribute/option/import_options.feature]: done

File: tests/legacy/features/pim/structure/attribute/option/edit_attribute_options.feature (hunk 2 above)
Comment (ahocquard) [reply tests/legacy/features/pim/structure/attribute/option/edit_attribute_options.feature]: all the tests are useless:
- successfully cancel edition of attribute options
- edition of an attribute option with a quote!
- display of the label is correctly impacted on the product: this is clearly testing the SPA and some cache we could have...

**Pull request closed without merging**